### ISSUE TITLE
large package upload slow

### ISSUE THREAD

**Issue by sleungcy** [cloudfoundry-community/terraform-provider-cloudfoundry#500]
bitsmanager is using clientV3.UploadPackage which does not take effect on skipping large files that are already in the cf cache. There are some improvement that can be made by using resource_matching, and skipping files under .cf_ignore.

I've a code change ready for this improvement and I can push it up for review soon.

**Comment by loafoe** [comment]
@sleungcy still an issue with latest release?

**Comment by sleungcy** [comment]
Yes, this is still an issue when deploying large artifacts, especially in the case of nodejs artifact with all node_modules bundled together as one single deployable unit.
